spec/rubocop/cop/style/hash_transform_values_spec.rb

```diff
@@ -81,6 +81,17 @@
     RUBY
   end
 
+  it 'flags _.to_h {...} when transform_values could be used' do
```

Comment:
I think it should necessary to detect it only for Ruby 2.6 or higher. It should not detected in Ruby 2.5 or lower.

```ruby
% ruby -v
ruby 2.5.8p224 (2020-03-31 revision 67882) [x86_64-darwin17]

% ruby -e "p({a: 1, b: 2, c: 3}.to_h { |k, v| [k, -v] })"
{:a=>1, :b=>2, :c=>3}

% ruby -e "p({a: 1, b: 2, c: 3}.transform_values { |v| -v })"
{:a=>-1, :b=>-2, :c=>-3}
```

Reply:
Fixed, thanks @koic! 🙇